metadata-db-app/src/cmr/metadata_db/services/concept_service.clj

```diff
@@ -177,33 +170,27 @@
     (when (> (count unknown-providers) 0)
       (errors/throw-service-error :not-found (msg/providers-do-not-exist unknown-providers)))))
 
-(defmulti set-or-generate-concept-id
+(defn set-or-generate-concept-id
```

Comment:
I think this func is worth creating a unit test for

Reply:
Yes, I agree.  I have created a test.